src/server/zset_family.cc

```diff
@@ -2396,37 +2396,61 @@ OpResult<MScoreResponse> ZSetFamily::ZGetMembers(CmdArgList args, ConnectionCont
 void ZSetFamily::GeoAdd(CmdArgList args, ConnectionContext* cntx) {
   string_view key = ArgS(args, 0);
 
-  // TODO: to handle options and multiple elements
   ZParams zparams;
+  size_t i = 1;
+  for (; i < args.size() - 1; ++i) {
```

Comment:
Out of curiosity, why `size() - 1`? I would understand `- 3` or (preferably) just `size()`

Reply:
correct, missed it!
I used the `size() - 1` because this is how it was handled in `ZADD`. Should I align here to `- 3` to keep it aligned?

Reply:
I'd use `size()` and return an error below (as I commented there)